core/templates/dev/head/pages/exploration_player/ExplorationPlayer.js

```diff
@@ -0,0 +1,24 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Data and controllers for the Oppia 'exploration_player' page.
+ */
+
+ oppia.controller('ExplorationPlayer', [
```

Comment:
Do we really need a space here?

Reply:
@DubeySandeep, what spaec are you referring to? AFAICT this matches the format in other files.

Reply:
One space before oppia.controller.

Reply:
Oh yes! Sorry, you're absolutely right. I thought you meant the space before the "[".